### DISCUSSION TITLE
duplicate Layout (paperspace) with all the configuration to another layout with a new name

### DISCUSSION THREAD

**Discussion by Marcelo5444** [mozman/ezdxf#633] [Q&A]
Hi, I have been looking in the documentation but have not found an easy way to duplicate a layout with a new name? Does that option exists?

**Comment by mozman** [comment]
`Ezdxf` has no such functionality.

**Comment by Marcelo5444** [comment]
From the top of your mind, is there an easy way of iterating trhough all the options of a layout and copying it to a new layout?
